djangocms_versioning/cms_config.py

```diff
@@ -25,6 +25,7 @@ class VersioningCMSExtension(CMSAppExtension):
 
     def __init__(self):
         self.versionables = []
+        self.add_to_context = collections.OrderedDict()
```

Comment:
On second thought, this probably doesn't need to be an OrderedDict as we only ever take one element of this dict per view.